src/pages/en/tutorial/1-setup/1.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: ~/layouts/TutorialLayout.astro
+title: Create a new project
+setup: |
+  import Badge from '~/components/Badge.astro';
+  import Checklist from '~/components/Checklist.astro';
+  import Goals from '~/components/tutorial/Goals.astro';
+---
+
+<Goals>
+  - run the `create-astro` setup wizard to install an Astro project in a local or cloud workspace
+  - started the Astro server in development (dev) mode
+  - viewed a live preview of your website in your browser
+  - made your first edit to your new website!
+</Goals>
+
+## Prepare your Development Environment
+
+This tutorial will include instructions for two types of development environments: ***local*** and ***cloud***.
+
+### 🖥️ Developing locally
+
+If you are **developing locally**, using files and applications that exist on your own computer, then your development environment includes **the terminal application** for your computer's operating system and your **code editor**. In this tutorial, we will use VS Code as the code editor. You will use a web browser to preview your work as you build.
+
+You will have a local copy of your files, and your repository on GitHub will be a copy kept in sync with your local project.
+
+### 🌐 Developing in the cloud
+
+If you are **developing in the cloud**, using StackBlitz or CodeSandbox, then your development environment is your **project workspace running in that browser tab**, which will provide a terminal console and an online version of VS Code in your browser. You will also have a simple browser pane to preview your work as you build. 
+
+There will be no local copy of your files, but there will be a working copy stored in your online account, which you will keep in sync with your source repository on GitHub.
+
+:::tip[working in the cloud?]
+There's no local development to set up! Your project will automatically run in development mode and display a website preview when you open the "Completely Empty" template from https://astro.new in StackBlitz or CodeSandbox. 
+
+**Next Steps**: 
+1. Click the button to "fork" the template (save to your own account dashboard).
+2. Skip ahead in this tutorial to [view a preview of your site](#view-a-preview-of-your-website).
+:::
+
+## Run the `create-astro` setup wizard
+
+The preferred way to create a new Astro site is through our `create-astro` setup wizard.
+
+1. In the command line of your terminal, run the command:
+`npm create astro@latest`
```

Comment:
flagging to switch to PackageManagerTabs

Reply:
Fine with this! I don't *think* there's anywhere later on that it really matters which package manager was used, so I have no issues showing all three... UNLESS we want to be opinionated (as we often are in this tutorial, for the sake of simplicity).

@delucis No rush, but if you are able to implement this before I'm back in my normal editing environment, please feel free to go ahead. (I'm not doing this myself while I'm only editing .md files directly on GitHub.com)